nix/haskell.nix

```diff
@@ -8,6 +8,7 @@
 , buildPackages
 , config ? {}
 # GHC attribute name
+## fixme: what is this for?
```

Comment:
It looks like this variable is unused.

Reply:
The `compiler` attribute should be removed. The compiler is supposed to be inferred fro either the stack.yaml or the cabal.project file. *if* we want to override it, we might want to provide `null` here and only use the override value if non-null.

Forcing `ghc-8.6.5` here will inevitably lead to issues if the stack.yaml (or project.cabal) specify a different compiler.

Reply:
OK thanks, removed.